internal/tiltfile/api/__init__.py

```diff
@@ -245,7 +245,7 @@ def docker_build(ref: str,
   """
   pass
 
-def docker_compose(configPaths: Union[str, Blob, List[Union[str, Blob]]], env_file: str = None, project_name: str = "", profiles: Union[str, List[str]] = []) -> None:
+def docker_compose(configPaths: Union[str, Blob, List[Union[str, Blob]]], env_file: str = None, project_name: str = "", profiles: Union[str, List[str]] = [], wait_for_healthy = False) -> None:
```

Comment:
what do you think of changing the param to `wait` to match the flag?

Reply:
That was my initial choice, actually. I was concerned in the context of Tilt it might not be obvious, but I'll chalk that up to "coding in the middle of the night," as it *literally* says `docker_compose` right there 😁

Reply:
sorry for the delay - this is done now